Add new API parameter and update observability docs (#1966)
vdaas/vald: docs/user-guides/observability-configuration.md

Vald publishes [the sample manifest](https://github.com/vdaas/vald/blob/main/charts/vald/values/dev-observability.yaml), which enables Grafana and Jaeger.

You can apply it after creating an observability environment.

<div class="notice">

> The default manifests don't set ingress host<BR>

Reviewer: ```suggestion
The default manifests don't set ingress host.<BR>
```

> you can monitor on the browser by port forwarding or set ingress host by yourself.

Reviewer: ```suggestion
You can monitor on the browser by port forwarding or set ingress host by yourself.
```

Author: ```suggestion
You can monitor the browser by port forwarding or define the ingress host by yourself.
```
